.github/actions/expo/channel/delete/action.yaml

```diff
@@ -23,5 +23,8 @@ runs:
       id: eas-update
       shell: bash
       run: |
-        eas channel:delete --non-interactive ${{ inputs.channel }}
-        eas branch:delete --non-interactive ${{ inputs.channel }}
+        eas channel:view ${{ inputs.channel }}  --non-interactive --limit 1
+        if [ $? -eq 0 ]; then
```

Comment:
you can also

```sh
if eas channel:view ${{ inputs.channel }}  --non-interactive --limit 1; then
```

Reply:
Which you'd need to do if you had `set -o errexit` set

Reply:
it seems like that may be implicitly set by the GH job runner, since my approach didn't work!

Reply:
thanks for the tip :)